Library/Homebrew/cask/dsl.rb

```diff
@@ -181,7 +181,11 @@ def url(*args, **options)
 
       set_unique_stanza(:url, args.empty? && options.empty? && !block_given?) do
         if block_given?
-          LazyObject.new { URL.new(*yield, from_block: true, caller_location: caller_location) }
+          LazyObject.new do
+            args = yield
```

Comment:
```suggestion
            *args = yield
```

Reply:
Otherwise `#last` will be called on e.g. a string, as seen in the failing audit currently.

Reply:
Thanks! Fixed